config/config.ci.edn

```diff
@@ -1,17 +1,17 @@
-{:db-spec       {:dbtype   "postgresql"
-                 :dbname   "chronograph_ci"
-                 :host     "localhost"
-                 :port     5432
-                 :user     "chronograph_ci_user"
-                 :password "chronograph_ci_pwd"}
- :oauth         {:google {:redirect-uri   "http://localhost:8000/google-oauth2-redirect"
-                          :response-type  "code"
-                          :login-endpoint "https://accounts.google.com/o/oauth2/v2/auth"
-                          :token-endpoint "https://oauth2.googleapis.com/token"
-                          :client-id      "test-client-id"
-                          :client-secret  "test-client-secret"
-                          :scope          "test scopes"}}
- :auth          {:token-signing-key       "your-signing-key-here"
-                 :token-expiry-in-seconds 3600}
- :app-log-level "debug"
- :port          8000}
+{:db-spec {:dbtype   "postgresql"
+           :dbname   "chronograph_ci"
+           :host     "localhost"
+           :port     5432
+           :user     "chronograph_ci_user"
+           :password "chronograph_ci_pwd"}
+ :oauth   {:google {:web-redirect-uri "http://localhost:8000/google-oauth2-redirect"
+                    :response-type    "code"
+                    :login-endpoint   "https://accounts.google.com/o/oauth2/v2/auth"
+                    :token-endpoint   "https://oauth2.googleapis.com/token"
+                    :client-id        "test-client-id"
+                    :client-secret    "test-client-secret"
+                    :scope            "test scopes"}}
+ :auth    {:token-signing-key           "your-signing-key-here"
+           :token-expiry-in-seconds     3600
+           :desktop-redirect-url-scheme "chronograph"}
+ :port    8000}
```

Comment:
What is the standard formatting/indentation convention? I very much prefer to _not_ vertically align k/v pairs because of the patch diff / review noise caused by minor indentation changes.

Reply:
I'm not opposed to that actually. I think we should just decide on a convention, configure https://github.com/weavejester/cljfmt and be done with it.

Reply:
However I do think aligned k/v pairs and let bindings are easier on the eyes. I don't think `cljfmt` supports this though.